packages/react/src/components/SearchField/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {
+  useState,
+  useRef,
+  ChangeEvent,
+  InputHTMLAttributes,
+  ReactNode,
+  forwardRef,
+  ForwardRefRenderFunction
+} from 'react';
+import classNames from 'classnames';
+
+import TextFieldWrapper from '../internal/TextFieldWrapper';
+import randomId from '../../utils/rndid';
+import Offscreen from '../Offscreen';
+import Icon from '../Icon';
+
+interface SearchFieldProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, 'onChange'> {
+  label: ReactNode;
+  value?: string;
+  defaultValue?: string;
+  onChange?: (value: string, e: ChangeEvent<HTMLInputElement>) => void;
+  hideLabel?: boolean;
+  isForm?: boolean;
+}
+
+const SearchField: ForwardRefRenderFunction<
```

Comment:
This is missing display name:

```js
SearchField.displayName = 'SearchField';
```

I think it might have to be set after wrapping this component with `forwardRef` instead of returning `forwardRef`.